### ISSUE TITLE
online! and offline! no longer work

### ISSUE THREAD

**Issue by caramdache** [colinta/motion-firebase#15]
```
(main)> Firebase.online!
2015-05-01 18:04:20.997 firebase[70331:2409939] firebase.rb:98:in `online!': undefined method `goOnline' for #<Firebase:0x11437e7e0> (NoMethodError)
=> #<NoMethodError: undefined method `goOnline' for #<Firebase:0x11437e7e0>>
(main)> Firebase.offline!
2015-05-01 18:04:30.683 firebase[70331:2409939] firebase.rb:91:in `offline!': undefined method `goOffline' for #<Firebase:0x11437e7e0> (NoMethodError)
=> #<NoMethodError: undefined method `goOffline' for #<Firebase:0x11437e7e0>>
```

**Comment by colinta** [comment]
Thanks!  I'll take a look when I have a second, but if you have time please investigate and submit a PR (if nothing else a failing spec file that calls these methods would help a lot).  I always appreciate the help!

**Comment by colinta** [comment]
Fixed in 3.1.5 :smiley:
